packages/client/src/components/Footer.jsx

```diff
@@ -7,23 +7,71 @@ import TwitterIcon from '@mui/icons-material/Twitter';
 import InstagramIcon from '@mui/icons-material/Instagram';
 import LinkedInIcon from '@mui/icons-material/LinkedIn';
 import YouTubeIcon from '@mui/icons-material/YouTube';
+import { Carousel } from './Carousel';
+import { useEffect, useRef, useState } from 'react';
 
 export const Footer = () => {
   const { loading, error, data } = useQuery(GET_LOGO_FULL_SVG);
+  const componentRef = useRef(null);
+  const [componentWidth, setComponentWidth] = useState(0);
+  const [displayedLogos, setDisplayedLogos] = useState(Math.floor((componentWidth - 120) / (200 + 40)));
```

Comment:
super nitpick: could you add a comment to say that 200 is Carousel's logoWidth and 40 is Carousel's logoMargin*2 :pray: 

And maybe do something like const calculateDisplayedLogos = Math.floor(blah) since this also appears later in the useEffect